variables.tf

```diff
@@ -118,19 +118,13 @@ variable "kube_vip_version" {
 
 variable "anthos_version" {
   type        = string
-  default     = "1.7.0"
+  default     = "1.8.3"
   description = "The version of Google Anthos to install"
 }
 
-variable "ccm_deploy_url" {
-  type        = string
-  default     = "https://gist.githubusercontent.com/thebsdbox/c86dd970549638105af8d96439175a59/raw/4abf90fb7929ded3f7a201818efbb6164b7081f0/ccm.yaml"
-  description = "The deploy url for the Equinix Metal CCM"
-}
-
 variable "kube_vip_daemonset_url" {
```

Comment:
This isn't used... And I think we should remove it and put this url in a "Local" variable similar to the way we did the CCM. So only the `Version` is the variable.

Reply:
I agree that the gist should be extensible. I like the idea of using a default string value that the user can override in a module with a `file("whatever they want")` pattern.

It doesn't seem necessary to treat this as a URL.

Reply:
We could take the approach of:
* If you gave us a string (`ccm_manifest`), we'll use it.
* If you gave us a URL (`ccm_manifest_url`), we'll fetch it via an `http_http` datasource. (we count use `count=0` when the string is set)
* We use a local variable to decide which variable to pass along to the templates.

My thoughts are somewhere between doing this and just setting:
```
variable "ccm_manifest" { 
  default = <<<EOS
BLAH
EOS
}```